python/ray/util/sgd/v2/trainer.py

```diff
@@ -142,7 +146,12 @@ def run(self,
             raise RuntimeError(
                 "This Trainer is not active. It is either shutdown already or "
                 "never started in the first place. Either create a new "
-                "Trainer or start this one.")
+                "Trainer or start this one.") from None
+        except SGDBackendError:
+            raise RuntimeError("Training failed. You should not be seeing "
+                               "this error and this is a bug. Please create "
+                               "a new issue at "
+                               "https://github.com/ray-project/ray.")
```

Comment:
raise from?

Let's try not to use raise without `from` when inside an except block -- it really clutters the output.

Reply:
yep good catch! this should be from None